packages/patternfly-4/content/design-guidelines/styles/charting_color_system.md

```diff
@@ -0,0 +1,39 @@
+path: /design-guidelines/styles/charting-color-system
```

Comment:
@gdoyle1 the path definition needs to be bracketed with `---` at top and bottom. Currently, the site cannot locate this page. You also need quotations around the path.

```
---
path: "/design-guidelines/styles/charting-color-system"
---
```

Reply:
how do I delete this? It's only supposed to be a path for "charts" in design-guidelines and then "colors-for-charts" in styles